content/en/docs/contribute/localization.md

```diff
@@ -68,82 +79,132 @@ contentDir = "content/de"
 weight = 3
 ```
 
-When assigning a `weight` parameter, see which of the current languages has the highest weight and add 1 to that value.
+When assigning a `weight` parameter for your block, find the language block with the highest weight and add 1 to that value.
+
+For more information about Hugo's multilingual support, see "[Multilingual Mode](https://gohugo.io/content-management/multilingual/)".
+
+### Add a new localization directory
 
-Now add a language-specific subdirectory to the [`content`](https://github.com/kubernetes/website/tree/master/content) folder. The two-letter code for German is `de`, so add a `content/de` directory:
+Add a language-specific subdirectory to the [`content`](https://github.com/kubernetes/website/tree/master/content) folder in the repository. For example, the two-letter code for German is `de`:
 
 ```shell
 mkdir content/de
 ```
 
+### Add a localized README 
+
+To guide other localization contributors, add a new [`README-**.md`](https://help.github.com/articles/about-readmes/) to the top level of k/website, where `**` is the two-letter language code. For example, a German README file would be `README-de.md`. 
+
+Provide guidance to localization contributors in the localized `README-**.md` file. Include the same information contained in `README.md` as well as:
+
+- A point of contact for the localization project 
+- Any information specific to the localization
+
+After you create the localized README, add a link to the file from the main English file, [`README.md`] and include contact information in English. You can provide a GitHub ID, email address, [Slack channel](https://slack.com/), or other method of contact.
```

Comment:
@ClaudiaJKang 👋 

> First ... If localization projects are managed within k/website repository, won't this might be a problem [with high amounts of repo traffic]?

There will be more traffic, yes. However, this PR introduces the ability for PR reviewers (on all teams, including English source review) to filter PRs by language labels. For example, any changes to files in `content/en/` receive an [automatically applied label](https://github.com/kubernetes/website/pull/10485/files#diff-32dd105d8a29047bc525448786218475R11): `language/en`. The same is true for other languages as well, with their respective labels.

> Second ... For example, does communication in Review or PR's title must be used English?

No. Localizations are free to work in their own idiom.

> Third ... Is this index's sequence right?

Yes. 👍 Do you have a better idea for organizing content?

> Fourth, how about adding link Localizing Kubernetes Documentation in README.md's Contributing to the docs section, followed by each localization's README-**.md link?

That's possible. For left nav links in Hugo, it would require moving the `README-**.md` files out of the top-level directory and into `content/<**>/docs/contribute/` with appropriate weighting.

Is the structural move something we could iterate in a follow-up PR?

Thanks for the feedback! ✨

Reply:
@zacharysarah Thank you for answering my review!

> Yes. +1 Do you have a better idea for organizing content?

Another idea to suggest is adding `README-**.md` example url in `localization.md` after this PR.

If so, localization team would be easier to create `README-**.md` configuration. 




> That's possible. For left nav links in Hugo, it would require moving the `README-**.md` files out of the top-level directory and into `content/<**>/docs/contribute/` with appropriate weighting.

I'm sorry but I think you've miss understood my fourth review. 

About the links at my fourth review are not left nav links but a `README-**.md`'s link.

I was mentioning, because localization team needs to add a link to `README-**.md` according to `localization.md` 

> After you create the localized README, add a link to the file from the main English file, [`README.md`] and include contact information in English.

So, how about adding `README-**.md` link in `README.md` like above?

> ## Contributing to the docs
> 
> ... contents ....
> 
> For more information about contributing to the Kubernetes documentation, see:
> 
> * [Start contributing](https://kubernetes.io/docs/contribute/start/)
> * [Staging Your Documentation Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
> * [Using Page Templates](http://kubernetes.io/docs/contribute/style/page-templates/)
> * [Documentation Style Guide](http://kubernetes.io/docs/contribute/style/style-guide/)
> * **[Localizing Kubernetes Documentation](https://kubernetes.io/docs/contribute/localization/) : [Chinese](https://github.com/kubernetes/website/README-cn.md) | [Korean](https://github.com/kubernetes/website/README-ko.md) | [Norsk](https://github.com/kubernetes/website/README-no.md)**